src/jquery.multiselect.js

```diff
@@ -118,18 +123,21 @@
 
       // The button that opens the widget menu.  Note that this is inserted later below.
       var $button = (this.$button = $( document.createElement('button') ) )
-            .addClass('ui-multiselect ui-widget ui-state-default ui-corner-all' + (classes ? ' ' + classes : ''))
+            .addClass('ui-multiselect ui-widget ui-state-default ui-corner-all'
+                        + (/\bbutton\b/i.test(wrapText) ? '' : ' ui-multiselect-nowrap')
+                        + (classes ? ' ' + classes : '')
+                        )
             .attr({
-              'type': 'button', 
-              'title': elSelect.title, 
-              'tabIndex': elSelect.tabIndex, 
-              'id': elSelect.id ? elSelect.id  + '_ms' : null 
+              'type': 'button',
+              'title': elSelect.title,
+              'tabIndex': elSelect.tabIndex,
+              'id': elSelect.id ? elSelect.id  + '_ms' : null
             })
             .prop('aria-haspopup', true)
             .html('<span class="ui-multiselect-open">' + iconSet.open + '</span>');    // Necessary to simplify dynamically changing the open icon.
 
       this.$buttonlabel = $( document.createElement('span') )
-            .html(options.noneSelectedText)
+            .html(options.noneSelectedText || $element[0].placeholder || 'Select options')
```

Comment:
Do you need the third item in the `||` clauses? The description made it sound like if the option was null it would use the placeholder. If it's null and there's no placeholder I think I would prefer it do nothing or log a warning instead of having two places for the nonSelectedText option in the code.

Reply:
That is fine.  I can take that out.